aws/resource_aws_lambda_layer_version_permission.go

```diff
@@ -0,0 +1,211 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAwsLambdaLayerVersionPermission() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsLambdaLayerVersionPermissionAdd,
+		Read:   resourceAwsLambdaLayerVersionPermissionGet,
+		Delete: resourceAwsLambdaLayerVersionPermissionRemove,
```

Comment:
change function to align with convention: *Create, *Read, *Delete

Reply:
functions names have been updated